<?xml version="1.0"?>

<launch>

  <!-- Note that axis IDs are those from the joystick message plus one, to be able to invert axes by specifiying either positive or negative axis numbers.-->
  <!-- Axis 2 from joy message thus has to be set as '3' or '-3'(inverted mode) below-->
  <node name="quadrotor_teleop" pkg="hector_quadrotor_teleop" type="quadrotor_teleop">
    <param name="x_axis" value="5"/>
    <param name="y_axis" value="4"/>
    <param name="z_axis" value="2"/>
    <param name="yaw_axis" value="1"/>
    <param name="x_velocity_max" value="30"/>
    <param name="y_velocity_max" value="-30"/>
    <param name="z_velocity_max" value="60000"/>
    <param name="yaw_velocity_max" value="-200"/>
  </node>
</launch>
